src/AzureClient/EntryPoint/EntryPointGenerator.cs

```diff
@@ -79,8 +79,9 @@ internal class EntryPointGenerator : IEntryPointGenerator
             return null;
         }
 
+        /// <inheritdoc/>
         public IEntryPoint Generate(string operationName, string? executionTarget,
-            RuntimeCapability? runtimeCapability = null)
+            RuntimeCapability? runtimeCapability = null, bool forceQir = false)
```

Comment:
nit: the last parameter could be named `generateQir` instead of `forceQir`.

Reply:
updated to use `generateQir`